test/browser/tests/survey.js

```diff
@@ -0,0 +1,19 @@
+const { click, button, write, into, textBox, waitFor } = require('taiko')
+
+// TODO it should be improved: wait for the loader div to disappear
+const waitForLoadComplete = async () => {
```

Comment:
this is a common utility function. extract it to some common place

Reply:
right, I'll move it elsewhere